pkg/util/kubeclient/ready.go

```diff
@@ -0,0 +1,172 @@
+package kubeclient
+
+import (
+	"context"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/openshift/openshift-azure/pkg/api"
+	"github.com/openshift/openshift-azure/pkg/util/ready"
+	"github.com/openshift/openshift-azure/pkg/util/wait"
+)
+
+var deploymentWhitelist = []struct {
+	Name      string
+	Namespace string
+}{
+	{
+		Name:      "docker-registry",
+		Namespace: "default",
+	},
+	{
+		Name:      "router",
+		Namespace: "default",
+	},
+	{
+		Name:      "registry-console",
+		Namespace: "default",
+	},
+	{
+		Name:      "customer-admin-controller",
+		Namespace: "openshift-infra",
+	},
+	{
+		Name:      "asb",
+		Namespace: "openshift-ansible-service-broker",
+	},
+	{
+		Name:      "webconsole",
+		Namespace: "openshift-web-console",
+	},
+	{
+		Name:      "console",
+		Namespace: "openshift-console",
+	},
+}
+
+var daemonsetWhitelist = []struct {
+	Name      string
+	Namespace string
+}{
+	{
+		Name:      "sync",
+		Namespace: "openshift-node",
+	},
+	{
+		Name:      "ovs",
+		Namespace: "openshift-sdn",
+	},
+	{
+		Name:      "sdn",
+		Namespace: "openshift-sdn",
+	},
+	{
+		Name:      "apiserver",
+		Namespace: "kube-service-catalog",
+	},
+	{
+		Name:      "controller-manager",
+		Namespace: "kube-service-catalog",
+	},
+	{
+		Name:      "apiserver",
+		Namespace: "openshift-template-service-broker",
+	},
+	{
+		Name:      "mdsd",
+		Namespace: "openshift-azure-logging",
+	},
+	{
+		Name:      "td-agent",
+		Namespace: "openshift-azure-logging",
+	},
+}
+
+var statefulsetWhitelist = []struct {
+	Name      string
+	Namespace string
+}{
+	{
+		Name:      "bootstrap-autoapprover",
+		Namespace: "openshift-infra",
+	},
+	{
+		Name:      "prometheus-forwarder",
```

Comment:
This was removed, remove forwarder at all

Reply:
ok thanks, i'll compare the file to the latest